### ISSUE TITLE
Updating from freeform 3 to 5. Keep getting field required error when uploading a file.

### ISSUE THREAD

**Issue by bartdigitalpulse** [solspace/craft3-freeform#1002]
Have a form with an input type field that is required. After selecting a file and hitting submit still get the required field error. 
The file gets uploaded in the assets lib. The response shows an asset id.
Tried both ajax and without. Setting the field as non required has a successfull submission.

The payload response:

```
{
    "success": false,
    "finished": false,
    "onSuccess": "reload",
    "id": null,
    "hash": "R1ZxAMxl2-noQ2ODb7e-LXmnX2jx8SHW8y8nlgR7jbANa9iCLeNyXPESQwjT",
    "values": {
        "firstName": "John",
        "lastName": "Smith",
        "email": "john@test.be",
        "phoneNumber": "0477555555",
        "communicationPreference": "Mail",
        "linkedIn": "sdqf",
        "vacancyId": "23394",
        "vacancy": "Digital M",
        "hoeHebJeOnsGevonden": "sd",
        "receiver": "",
        "curriculumVitae": [
            23473
        ]
    },
    "errors": {
        "curriculumVitae": [
            "Dit veld is verplicht"
        ]
    },
    "formErrors": [],
    "returnUrl": "https://test.ddev.site/nl/sollicitatie-succes",
    "submissionId": null,
    "submissionToken": "P3aABMhzoLHaxJ0taJov0Lvtt2QaFRZYcxVODW1qGRVa0ufbbgLUpndmxZhLh9KOQkJWEzatByexTzEpynXwMtvuv8n1HpAn9DR5",
    "html": "<removed>",
    "actions": [],
    "multipage": false,
    "duplicate": false,
    "freeform_payload": "<removed>"
}
```

**Steps to reproduce**

1. From with required file field
2. Select file
3. Submit

**Expected behavior**
No required error

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.7.1
- Freeform Version: 5.0.5
- Freeform Edition:Pro
- Fresh Install or Upgrade: Upgrade

**Comment by gustavs-gutmanis** [comment]
Sorry for the inconveniece @bartdigitalpulse 
And thanks for reporting this.

We will release a fix for this later today.

**Comment by kjmartens** [comment]
This is now resolved in Freeform 5.0.6+ 🙂
